src/data/taskList.ts

```diff
@@ -26,8 +28,12 @@ export const initialReviewTag = (intakeStatus: string) => {
     : 'CANNOT_START';
 };
 
-export const businessCaseTag = (intakeStatus: string) => {
-  switch (intakeStatus) {
+export const businessCaseTag = (intake: SystemIntakeForm) => {
+  if (intake.requestType === 'RECOMPETE') {
+    return 'NOT_NEEDED';
```

Comment:
I thought the status of these steps was "cannot start" until the intake was decided on? I also jumped into the design huddle a little late today, so I defer to your memory here.

Reply:
you right, good catch!